package.json

```diff
@@ -239,7 +239,7 @@
 		"clean:packages": "rimraf \"./packages/*/@(build|build-module|build-style)\"",
 		"clean:package-types": "tsc --build --clean",
 		"prebuild:packages": "npm run clean:packages && lerna run build",
-		"build:packages": "npm run build:package-types && node ./bin/packages/build.js",
+		"build:packages": "node ./bin/packages/build.js",
```

Comment:
`npm run build:package-types` is removed here, so `tsc --build` is also not run. I think we could just add `tsc --build` here though to make it explicit.

Reply:
ayyyy, forgot about this, it was a temporary change because the types were broken. Thanks :)